plugins/PrerenderPlugin/index.js

```diff
@@ -1,10 +1,33 @@
 const fs = require('fs');
 const path = require('path');
+const gracefulFs = require('graceful-fs');
 const chalk = require('chalk');
 const {SyncHook} = require('tapable');
+const {htmlTagObjectToString} = require('html-webpack-plugin/lib/html-tags');
 const templates = require('./templates');
 const vdomServer = require('./vdom-server-render');
 
+const preRenderPluginHooksMap = new WeakMap();
```

Comment:
Should this name be `preRender...`? or can be changed to `prerender...`?

Reply:
🆗